README.md

```diff
@@ -8,72 +8,228 @@
 [![Slack](https://img.shields.io/badge/slack-%23humbug-red.svg?style=flat-square)](https://symfony.com/slack-invite)
 [![License](https://img.shields.io/badge/license-MIT-red.svg?style=flat-square)](LICENSE)
 
-PHP-Scoper is a tool for adding a prefix to all PHP namespaces in a given file
-or directory. 
-
+PHP-Scoper is a tool which essentially moves any body of code, including all
+dependencies such as vendor directories, to a new and distinct namespace.
 
 ## Goal
 
-PHP-Scoper's goal is to make sure that all code in a directory lies in a 
-distinct PHP namespace. This is necessary when building PHARs that 
+PHP-Scoper's goal is to make sure that all code for a project lies in a 
+distinct PHP namespace. This is necessary, for example, when building PHARs that:
 
-* Bundle their own vendor dependencies
-* Load code of arbitrary PHP projects
+* Bundle their own vendor dependencies; and
+* Load/execute code from arbitrary PHP projects with similar dependencies
 
 These PHARs run the risk of raising conflicts between their bundled vendors and 
-the vendors of the loaded project, if the vendors are required in incompatible
-versions.
-
+the vendors of the project it is interacting with, where the PHAR's dependencies
+are used preferentially leading to difficult to debug issues due to dissimilar or
+unsupported package versions.
 
 ## Installation
 
-You can install PHP-Scoper with Composer:
+The preferred method of installation is to use the PHP-Scoper phar, which can
+be downloaded from the most recent Github Release. Subsequent updates can be
+downloaded by running:
 
 ```bash
-composer global require humbug/php-scoper:dev-master
+php-scoper.phar self-update
 ```
 
-If you cannot install it because of a dependency conflict or you prefer to
```

Comment:
I would divide the installation in two parts: "PHAR (preferred)" and "Composer". Hence keeping the paragraph below still just in case. We could remove the `:dev-master` part though